Image build fail (lagoon/php:7.2-fpm)

**Describe the bug**
When building the container images locally, via `make build` the `php:7.2-fpm-alpine3.11` image fails to build successfully. The build step is adding curl package via APK.

**To Reproduce**
Steps to reproduce the behavior:
1. run `make build` on current master branch
2. wait

**Expected behavior**
Image should be built successfully

**Screenshots**

```Removing intermediate container 7529952b24ae
 ---> 1829778235f0
Step 24/31 : RUN apk add --no-cache curl=7.69.1-r0 libcurl=7.69.1-r0 --repository http://dl-cdn.alpinelinux.org/alpine/edge/main/
 ---> Running in 45601728d295
fetch http://dl-cdn.alpinelinux.org/alpine/edge/main/x86_64/APKINDEX.tar.gz
fetch http://dl-cdn.alpinelinux.org/alpine/v3.11/main/x86_64/APKINDEX.tar.gz
fetch http://dl-cdn.alpinelinux.org/alpine/v3.11/community/x86_64/APKINDEX.tar.gz
ERROR: unsatisfiable constraints:
  libcurl-7.67.0-r0:
    breaks: world[libcurl=7.69.1-r0]
    satisfies: world[so:libcurl.so.4]
               curl-7.67.0-r0[so:libcurl.so.4]
  curl-7.67.0-r0:
    breaks: world[curl=7.69.1-r0]
The command '/bin/sh -c apk add --no-cache curl=7.69.1-r0 libcurl=7.69.1-r0 --repository http://dl-cdn.alpinelinux.org/alpine/edge/main/' returned a non-zero code: 2
Makefile:287: recipe for target 'build/php__7.2-fpm' failed
make: *** [build/php__7.2-fpm] Error 2
```

Managed to reproduce the problem locally:

```
make clean
make -j4 build/php__7.4-cli
```

Error

```
 ---> Running in ad224cf53a20
fetch http://dl-cdn.alpinelinux.org/alpine/edge/main/x86_64/APKINDEX.tar.gz
fetch http://dl-cdn.alpinelinux.org/alpine/v3.11/main/x86_64/APKINDEX.tar.gz
fetch http://dl-cdn.alpinelinux.org/alpine/v3.11/community/x86_64/APKINDEX.tar.gz
ERROR: unsatisfiable constraints:
  libcurl-7.67.0-r0:
    breaks: world[libcurl=7.69.1-r0]
    satisfies: world[so:libcurl.so.4]
               curl-7.67.0-r0[so:libcurl.so.4]
  curl-7.67.0-r0:
    breaks: world[curl=7.69.1-r0]
The command '/bin/sh -c apk add --no-cache curl=7.69.1-r0 libcurl=7.69.1-r0 --repository http://dl-cdn.alpinelinux.org/alpine/edge/main/' returned a non-zero code: 2
make: *** [build/php__7.4-fpm] Error 2
```

I checked http://dl-cdn.alpinelinux.org/alpine/edge/main/x86_64/ and it looks like `7.69.1-r0` is no longer there (replaced by `7.70.0-r0`). Also checked Alpine Linux 3.11, and still it is running the old (buggy) version.

--- a/images/php/fpm/Dockerfile
+++ b/images/php/fpm/Dockerfile
@@ -45,7 +45,7 @@ ENV NEWRELIC_VERSION=9.10.1.263
 # @see https://github.com/curl/curl/issues/4624
 # @see https://pkgs.alpinelinux.org/packages?name=curl&branch=v3.11
 # TODO: Remove as soon as Alpine 3.11 is shipped with a version higher than 7.67.0
-RUN apk add --no-cache curl=7.70.0-r0 libcurl=7.70.0-r0 --repository http://dl-cdn.alpinelinux.org/alpine/edge/main/
+RUN apk add --no-cache curl --repository http://dl-cdn.alpinelinux.org/alpine/edge/main/
 
 RUN apk add --no-cache fcgi \
         ssmtp \

--- a/images/php/fpm/Dockerfile
+++ b/images/php/fpm/Dockerfile
@@ -41,10 +41,11 @@ COPY ssmtp.conf /etc/ssmtp/ssmtp.conf
 # @see https://docs.newrelic.com/docs/agents/php-agent/getting-started/php-agent-compatibility-requirements
 ENV NEWRELIC_VERSION=9.10.1.263
 
-# Pin curl to Version 7.69.1-r0 as the current shipped one 7.67.0 has a bug, see
-# https://github.com/curl/curl/issues/4624
+# Pin curl to Version 7.70.0-r0 as the current shipped one 7.67.0 has a bug.
+# @see https://github.com/curl/curl/issues/4624
+# @see https://pkgs.alpinelinux.org/packages?name=curl&branch=v3.11
 # TODO: Remove as soon as Alpine 3.11 is shipped with a version higher than 7.67.0
-RUN apk add --no-cache curl=7.69.1-r0 libcurl=7.69.1-r0 --repository http://dl-cdn.alpinelinux.org/alpine/edge/main/
+RUN apk add --no-cache curl=7.70.0-r0 libcurl=7.70.0-r0 --repository http://dl-cdn.alpinelinux.org/alpine/edge/main/
 
 RUN apk add --no-cache fcgi \
         ssmtp \

--- a/images/php/fpm/Dockerfile
+++ b/images/php/fpm/Dockerfile
@@ -41,10 +41,6 @@ COPY ssmtp.conf /etc/ssmtp/ssmtp.conf
 # @see https://docs.newrelic.com/docs/agents/php-agent/getting-started/php-agent-compatibility-requirements
 ENV NEWRELIC_VERSION=9.10.1.263
 
-# Pin curl to Version 7.70.0-r0 as the current shipped one 7.67.0 has a bug.
-# @see https://github.com/curl/curl/issues/4624
-# @see https://pkgs.alpinelinux.org/packages?name=curl&branch=v3.11
-# TODO: Remove as soon as Alpine 3.11 is shipped with a version higher than 7.67.0
 RUN apk add --no-cache curl --repository http://dl-cdn.alpinelinux.org/alpine/edge/main/
 
 RUN apk add --no-cache fcgi \

